Feat188 patterns weaves (#192)
federated-research/docs: website/pages/federated_research_patterns/computational/pooled.mdx

> #  Pooled Pattern

Reviewer: I'm concerned about using the term 'pattern' for this, as we are (or were) using pattern to mean things like the analytical and egress patterns. Are we now meaning that the patterns are computational, egress, platform, and consist of a combination of components?

If so, I think what we mean by pattern should be captured somewhere.

Author: There are computational, egress, platforms patterns. 

In each area there is then a specific pattern.

**Analytical:** Centralised <br/>
**Data Movement**: Row level data <br/>

> **Required Trust between TREs**: Very High <br/>

Reviewer 2: Is `Required Trust between TREs` a Component? I ask as it's given the same weight as the other components here and on the other computational pattern pages, but not listed as a component in the computational page, which is inconsistent.

I wonder if this is a design problem, and we could have a way of making it clear to a reader what the components are (future work).

Author: Yeah, maybe adding in too much too soon, will remove for now, and add on backlog.

> The pattern captures the movement of data where row level data moves, either persistently or temporarily to perform an analysis. Outside of TREs this is the most common method, where researchers would be accustomed to data being on their data where they can perform any analysis or learning. 

Reviewer: there seems to be an error in the text here.

Author: grammar or something else?

> This approach requires a very high level of trust between TREs as it requires sharing of row-level data between TREs. Disclosure control is still applied before the data leaves the TRE network. 

Reviewer: Going to suggest a minor change here - disclosure control *can* still be applied, because really it's up to the TRE. "None" is a valid egress control.

```suggestion
This approach requires a very high level of trust between TREs as it requires sharing of row-level data between TREs. Disclosure control can still be applied before the data leaves the TRE network. 
```

> This approach does require a very high level of trust between TREs as it would require effective row level sharing of data between TREs. Disclosure control would still be applied before the data left the TRE network. 

Reviewer 3: There's quite a bit of mixing tenses here. Do you want it to be "this pattern **is**" or "this pattern **would be**"?

```suggestion
This approach requires a very high level of trust between TREs as it requires sharing of row-level data between TREs. Disclosure control is still applied before the data leaves the TRE network. 
```

![Alt Text](/assets/gif/Dynamics.gif)

## Variance in implementation

> An emerging pattern from the DARE UK TREvolution programme is that a project specific network is created between the TREs, to either move or provide the researcher with a view of the data that makes it appear as if the data is all in one location. This approach is by far the most powerful for a user and the analyses and learning are almost unlimited.

Reviewer 4: ```suggestion
An emerging pattern from the DARE UK TREvolution programme is that a project-specific network is created between the TREs, to either move or provide the researcher with a view of the data that makes it appear as if the data is all in one location. This approach is by far the most powerful for a user and the analyses and learning are almost unlimited.
```

> An emerging pattern from the DARE UK TREvolution programme is that a project specific network is created between the TREs, to either move or provide the researcher with a view of the data that makes it appear as if the data is all in one location. This approach is by far the most powerful for a user and the analyses and learning are almost unlimited.

Reviewer 3: Is this an emerging pattern, or are you talking about a particular implementation?

Also "analyses and learning are almost unlimited" - is the learning machine learning? I would be more specific and go with something like "possibilities for analysis and machine learning are almost unlimited"

I might even step it back a bit and say they're (almost?) equivalent to really having the data in one place
